src/screens/runPlan/runPlanList/PlanItemListItem.tsx

```diff
@@ -1,9 +1,9 @@
 import React from 'react';
-import { StyleSheet, TouchableHighlight } from 'react-native';
+import {StyleSheet, TouchableHighlight, ViewStyle} from 'react-native';
 
 import { Card } from 'components';
 import { PlanItem } from 'models';
-import { palette } from 'styles';
+import {palette, typography} from 'styles';
```

Comment:
Is it a differenece when spaces are used or omitted at the ends of { }?

Reply:
Not a difference - PyCharm put imports in this way by default :)